sleepecg/feature_extraction.py

```diff
@@ -471,13 +471,20 @@ def preprocess_rri(
     -------
     np.ndarray
         The cleaned RRI series.
+
+    Examples
+    --------
+    Mask RR intervals outside the range of 0.4 to 2 seconds (= 30 to 150 bpm)
+
+    >>> from sleepecg import preprocess_rri
+    >>> preprocess_rri([0.5, 0.2, 0.8, 2.5, 0.6], min_rri=0.4, max_rri=2)  # noqa
```

Comment:
why do we need the `# noqa` here?

Reply:
To avoid flake8 warning:

![image](https://user-images.githubusercontent.com/25703792/169740882-d017ae61-e734-4318-ae55-81959d235cff.png)

But feel free to remove!

Reply:
Ah, I see, that should be the "Mask RR intervals [...]" line, right? So I'd remove this here and shorten the other line a bit :)

```suggestion
    >>> preprocess_rri([0.5, 0.2, 0.8, 2.5, 0.6], min_rri=0.4, max_rri=2)
```